src/frontend/js/utils/StringHelper.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { StringHelper } from './StringHelper';
+
+describe('StringHelper', () => {
+  it('is a string', () => {
+    expect(StringHelper.isString('Hello')).toBeTruthy();
```

Comment:
Use `toBe(value)` instead of `toBeTruthy` or `toBeFalsy`. It is safer to limit the risk of false positive.

```suggestion
    expect(StringHelper.isString('Hello')).toBe(true);
```